src/main/java/org/mskcc/cmo/publisher/CmoMetaDbPublisherPipeline.java

```diff
@@ -0,0 +1,95 @@
+        CommandLine commandLine = parseArgs(args);
+
+        String jobName = null;
+        JobExecution jobExecution = null;
+        if (commandLine.hasOption("r") || commandLine.hasOption("s")) {
+            jobName = BatchConfiguration.LIMS_REQUEST_PUBLISHER_JOB;
+            jobExecution = launchLimsRequestPublisherJob(ctx, commandLine.getOptionValue("r"),
+                    commandLine.getOptionValue("s"), commandLine.getOptionValue("e"));
+        } else {
+            LOG.error("Must run application with at least option '-r' or '-s' - exiting...");
+            System.exit(1);
+        }
+        if (!jobExecution.getExitStatus().equals(ExitStatus.COMPLETED)) {
+            LOG.error(jobName + " failed with exit status: " + jobExecution.getExitStatus());
+            System.exit(1);
+        }
+    }
+
+    private static JobExecution launchLimsRequestPublisherJob(ConfigurableApplicationContext ctx,
+            String requestIds, String startDate, String endDate) throws Exception {
+        JobParameters jobParameters = new JobParametersBuilder()
+                .addString("requestIds", requestIds)
+                .addString("startDate", startDate)
+                .addString("endDate", endDate)
+                .toJobParameters();
+        JobLauncher jobLauncher = ctx.getBean(JobLauncher.class);
+        Job job = ctx.getBean(BatchConfiguration.LIMS_REQUEST_PUBLISHER_JOB, Job.class);
+        return jobLauncher.run(job, jobParameters);
+    }
+
+    private static Options getOptions(String[] args) {
+        Options options = new Options();
+        options.addOption("h", "help", false, "shows this help document and quits.")
+                .addOption("r", "request_ids", true, "Comma-separated list of request ids to fetch "
+                + "data for from LimsRest")
+                .addOption("s", "start_date", true, "Start date [YYYY/MM/DD], fetch requests from "
+                        + "LimsRest beginning from the given start date")
+                .addOption("e", "end_date", true, "End date [YYYY/MM/DD]. Fetch requests from LimsRest "
+                        + "between the start and end dates provided. [OPTIONAL]");
+        return options;
+    }
+
+    private static void help(Options options, int exitStatus) {
+        HelpFormatter helpFormatter = new HelpFormatter();
+        helpFormatter.printHelp("CmoNewRequestPublisher", options);
+        System.exit(exitStatus);
+    }
+
+    private static CommandLine parseArgs(String[] args) throws Exception {
+        Options options = getOptions(args);
+        CommandLineParser parser = new DefaultParser();
+        CommandLine commandLine = parser.parse(options, args);
+        if (commandLine.hasOption("h")
+                || (!commandLine.hasOption("r") && !commandLine.hasOption("s"))) {
+            help(options, 0);
+        }
+        if (commandLine.hasOption("r") && (commandLine.hasOption("s") || commandLine.hasOption("e"))) {
```

Comment:
I think you can do more validation here - "s" and "e" (if passed) contain valid date formats.  Also that "e" comes after "s".